### ISSUE TITLE
forceRefocus Not working

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1180]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.10](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.9...3.1.10) (2020-12-14)

- build(deps-dev): bump @types/node from 14.14.12 to 14.14.13 [`#1179`](https://github.com/open-wa/wa-automate-nodejs/pull/1179)
- build(deps-dev): bump husky from 4.3.5 to 4.3.6 [`#1178`](https://github.com/open-wa/wa-automate-nodejs/pull/1178)
- Release 3.1.10 [`cccfc0d`](https://github.com/open-wa/wa-automate-nodejs/commit/cccfc0d7db28015b0b1c70316603ab6573bc9cf3)
- fix: forceRefocus now working as expected #1180 [`dfbff1d`](https://github.com/open-wa/wa-automate-nodejs/commit/dfbff1d82736d3ded2682b5ddf2dcb0730eda681)
